README.md

```diff
@@ -1,6 +1,6 @@
 # pandas-mentoring
 <img src="img/logo.jpeg" width="600" height="200"><br>
-
+	
```

Comment:
Do you want to revert this?

Surely not a problem at this point, but in projects like pandas, unrelated changes should be avoid (even if they aren't wrong, they change the git history, and we prefer to keep things under control by being strict at those things).

Reply:
Sure! Do you have a suggestion on how to revert this?

Reply:
the "correct" way is probably some sort of `git reset`, but I think it's better in this case to simply edit the file, remove the spaces in that line, and add a new commit to this PR